src/python/WMCore/WorkQueue/Policy/Start/Block.py

```diff
@@ -227,8 +227,13 @@ def getMaskedBlocks(self, task, dbs, datasetPath):
             
             for slumis in grouper(lumis, 1000):
                 for block in blocks:
-                    slicedFiles = dbs.dbs.listFileArray(block_name=block.get('block_name'), run_num=run,
-                                                        lumi_list=slumis, detail=True)
+                    # run_num=1 will not be allowed anymore in this API (only for LFNs, not blocks
+                    # nor dataset), starting on Feb/2016
+                    if run==1:
+                        slicedFiles = dbs.dbs.listFileArray(block_name=block.get('block_name'), lumi_list=slumis, detail=True)
```

Comment:
Don't worry Seangchan. Yes, I'm running a test against pre5 and found a couple of issues already, still a few more to sort out. I also want to test lfns with more than one run number, the current patch would break.

Maybe I can drop an email to the HTTP group and say we'll deliver new WMCore tags only tomorrow afternoon. It should not be an issue and our crab friends to it all the time :-D